tests/test_raw_output_matches_git_on_full_repo_history

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # if first arg provided, use it, otherwise default to release
-DELTA_BIN=${1:-./target/release/delta}
+DELTA_BIN=${1:=./target/release/delta}
```

Comment:
This change is here to make it fail deliberately; I will revert this before merging.